### ISSUE TITLE
MySQL 5.6 compatibility

### ISSUE THREAD

**Issue by wdalmut** [cachethq/Cachet#319]
I don't know if mysql 5.6 is in the compatibility list but with that version of mysql during migrations i receive that error

```
Migration table created successfully.
                                                                                                                                                                                              
  [Illuminate\Database\QueryException]                                                                                                                                                         
  SQLSTATE[42000]: Syntax error or access violation: 1101 BLOB/TEXT column 'description' can't have a default value (SQL: create table `components` (`id` int unsigned not null auto_incremen  
  t primary key, `name` varchar(255) not null, `description` text not null default '', `link` text not null default '', `tags` text not null default '', `status` int not null, `order` int n  
  ot null, `group_id` int not null, `user_id` int not null, `created_at` timestamp default 0 not null, `updated_at` timestamp default 0 not null, `deleted_at` timestamp null) default charac  
  ter set utf8 collate utf8_unicode_ci)                                                         
  [PDOException]                                                                                                     
  SQLSTATE[42000]: Syntax error or access violation: 1101 BLOB/TEXT column 'description' can't have a default value  
                                                                   
migrate [--bench[="..."]] [--database[="..."]] [--force] [--path[="..."]] [--package[="..."]] [--pretend] [--seed]
```

With mysql 5.5 all works correctly.

I tried with docker pulling the mysql:5.6 repository

```
docker run --name cachet-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=root -d mysql:5.6
```

**Comment by jbrooksuk** [comment]
Can you try running the latest version 873f726671473bf1e06f31ca7c65cdc0bf2c927b please.

FWIW I run MySQL 5.6 locally for development and have had no issues. We've recently rewritten all of the migrations too, so that may help.

**Comment by Crash--** [comment]
I'm also running 5.6(.22) locally, and had encounter no issues with migrations today.

**Comment by wdalmut** [comment]
I have always the same issue. Here is the whole error:

```
13:02 $ docker run -e MYSQL_ROOT_PASSWORD=root --link cachet-mysql:cachet -it mysql:5.6 mysql -uroot -proot -hcachet                                                                                                                                                           
Warning: Using a password on the command line interface can be insecure.
Welcome to the MySQL monitor.  Commands end with ; or \g.
Your MySQL connection id is 3
Server version: 5.6.22 MySQL Community Server (GPL)

Copyright (c) 2000, 2014, Oracle and/or its affiliates. All rights reserved.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

mysql> create database cachet;
Query OK, 1 row affected (0.00 sec)

mysql> exit
Bye
✔ ~/Downloads/Cachet [master|✚ 12…1] 
13:02 $ ./artisan migrate
Migration table created successfully.
                                                                                                                                                                                                                                                                              
  [Illuminate\Database\QueryException]                                                                                                                                                                                                                                        
  SQLSTATE[42000]: Syntax error or access violation: 1101 BLOB/TEXT column 'description' can't have a default value (SQL: create table `components` (`id` int unsigned not null auto_increment primary key, `name` varchar(255) not null, `description` text not null defaul  
  t '', `link` text not null default '', `tags` text not null default '', `status` int not null, `order` int not null, `group_id` int not null, `user_id` int not null, `created_at` timestamp default 0 not null, `updated_at` timestamp default 0 not null, `deleted_at` t  
  imestamp null) default character set utf8 collate utf8_unicode_ci)                                                                                                                                                                                                          
                                                                                                                     
  [PDOException]                                                                                                     
  SQLSTATE[42000]: Syntax error or access violation: 1101 BLOB/TEXT column 'description' can't have a default value

migrate [--bench[="..."]] [--database[="..."]] [--force] [--path[="..."]] [--package[="..."]] [--pretend] [--seed]
```

Instead, the same procedure with mysql 5.5.41 goes correctly without problems.

**Comment by jbrooksuk** [comment]
@wdalmut are you using the latest version?

**Comment by jbrooksuk** [comment]
Locally I use MySQL 5.6.21 and have had no issues with the migrations currently. I can only assume your using old migrations.

**Comment by wdalmut** [comment]
I am using  mysql 5.6.22 in a docker container (the official image)

Just try with:

```
docker run --name cachet-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=root -d mysql:5.6
```

connect to local MySQL using `root` with password `root`

**Comment by jbrooksuk** [comment]
Fixed by 1d54d7553f1d69e68504cb7e492c2c5ad7726b99.
